### PR TITLE
Added arrest data

### PR DESCRIPTION
**Pull request by brandonp728**
All arrest data denoted by red gavel icon

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -108,6 +108,12 @@
             iconSize: [32, 32],
             iconAnchor: [16, 32]
         }),
+		CITY_ARREST: L.divIcon({
```
Comment (vonbearshark) [inline main.js]: is this indentation off, or is that just GH?

File: main.js [hunk 2]
```diff
@@ -231,6 +256,7 @@
     Promise.all([
         fetchWPRDCData('Police', { limit: 250 }),
         fetchWPRDCData('311', { limit: 250 }),
+		fetchWPRDCData('Arrest', { limit: 250 }),
```
Comment (vonbearshark) [inline main.js]: Likewise, is the indentation off here?

**Review by vonbearshark — COMMENTED** [review]
Wow, that was fast! Looks like some slight indentation changes, then we'll be good!

**Comment by vonbearshark** [comment]
I was worried this set overlaps with the city police set, but that's clearly not the case. Lots of interesting data here, especially underage liquor arrests.

File: main.js (hunk 2 above)
Comment (brandonp728) [reply main.js]: Thanks! Didn't notice them in my editor, all good now.

**Review by vonbearshark — APPROVED** [review]
Looks perfect!

**Pull request merged**